### DISCUSSION TITLE
適合的模型提問

### DISCUSSION THREAD

**Discussion by XiaXia009** [PaddlePaddle/PaddleOCR#14062] [Q&A]
想問這種字體甚麼模型辨識效果會比較好🙇
![image](https://github.com/user-attachments/assets/4ba34e79-5f9b-4a7b-9d95-1120a3b3e4a7)
![image](https://github.com/user-attachments/assets/ad709cd6-d0ba-4664-8850-bd36a627c9ea)
![image](https://github.com/user-attachments/assets/7ed1001a-cb6c-4b05-b807-101b256b4b9c)

**Comment by GreatV** [comment]
估计都不行吧，可能需要你自己训练一下

**Comment by GreatV** [comment]
https://paddlepaddle.github.io/PaddleOCR/latest/ppocr/model_list.html 先从这里找找试试看
